dev/io.openliberty.http.monitor/bnd.bnd

```diff
@@ -25,7 +25,8 @@ src: src
 
 -dsannotations: \
   io.openliberty.http.monitor.ServletContainerInitializer,\
-  io.openliberty.http.monitor.metrics.RestMetricManager
+  io.openliberty.http.monitor.metrics.RestMetricManager,\
+  io.openliberty.http.monitor.MonitorAppStateListener
 
 WS-TraceGroup: 
```

Comment:
I have no idea why you have your Import-Package and DynamicImport-Package the way you do for this bundle.  It would seem you should be able to use normal Import-Package and not need to have the dynamic import at all.  I would remove Import-Package and DynamicImport-Package completely and see how things go.

Reply:
Ah that was left over from the prototyping when this wasn't an auto-feature and I was testing dynamically switching features in and out.

Updated to the following, still need to dynamically import this package other wise the probes won't fire for some reason.

```
Import-Package: \
    !com.ibm.ws.http.*,\
    *
    
DynamicImport-Package: \
  com.ibm.ws.http.*
```

Reply:
I am going to mark resolved, but I am interested in why it does this and what error you get.

Reply:
It doesn't throw any errors or ffdcs. But through the debugger, when it runs this line
```
HttpDispatcherLink httpDispatcherLink = (HttpDispatcherLink)probedHttpDispatcherLinkObj; 
^ (line 182)
```
The monitor-1.0 _probe_ related code encounterse a `NoClassDefFoundError`